README.md

```diff
@@ -185,27 +185,9 @@ specifying target platform. In addition, Buildx also supports new features that
 are not yet available for regular `docker build` like building manifest lists,
 distributed caching, and exporting build results to OCI image tarballs.
 
-Buildx is supposed to be flexible and can be run in different configurations
-that are exposed through a driver concept. Currently, we support:
-
-- a [`docker` driver](docs/reference/buildx_create.md#docker-driver) that uses
-  the BuildKit library bundled into the Docker daemon binary,
-- a [`docker-container` driver](docs/reference/buildx_create.md#docker-container-driver)
-  that automatically launches BuildKit inside a Docker container,
-- a [`kubernetes` driver](docs/reference/buildx_create.md#kubernetes-driver) to
-  spin up pods with defined BuildKit container image to build your images.
-- a [`remote` driver](docs/reference/buildx_create.md#remote-driver) to
-  connect to manually provisioned and managed buildkitd instances.
-
-We plan to add more drivers in the future.
-
-The user experience of using buildx is very similar across drivers, but there
-are some features that are not currently supported by the `docker` driver,
-because the BuildKit library bundled into docker daemon currently uses a
-different storage component. In contrast, all images built with `docker` driver
-are automatically added to the `docker images` view by default, whereas when
-using other drivers the method for outputting an image needs to be selected
-with `--output`.
+Buildx is flexible and can be run in different configurations that are exposed
+through various "drivers". For more information, see the
+[drivers guide](docs/guides/drivers/index.md).
```

Comment:
I think it might be beneficial to keep a small intro of drivers in the main readme and then link to the drivers guide. Otherwise, reading docs about `buildx create` is quite confusing if the reader doesn't have a basic understanding of what drivers exist.

Reply:
Have kept a bit of explanation of drivers, left the list of drivers, and kept it similar to the index.md with links to guides/references.